app/src/main/java/org/thoughtcrime/securesms/preferences/PrivacySettingsPreferenceFragment.kt

```diff
@@ -87,7 +87,7 @@ class PrivacySettingsPreferenceFragment : ListSummaryPreferenceFragment() {
                     }
                         .apply { addFlags(Intent.FLAG_ACTIVITY_NEW_TASK) }
                         .takeIf { IntentUtils.isResolvable(requireContext(), it) }.let {
-                        startActivity(it)
+                        if (it != null) startActivity(it)
```

Comment:
Can `it` ever be null if you are inside the `let`?

Reply:
"it"'s type is `Intent?` so I don't think it hurts to check.

Reply:
But is it not achieved by using that `let`? Is there a case where let somehow lets through a null value?

Reply:
I guess no, it's not possible for the intent to be null since we create the intent on line 85 and then chain .apply -> .takeif -> .let to it. Check removed.

Reply:
Looks like it's still there

Reply:
Apologies, I did remove it - but then I stashed some code. Check removed for real this time.